repose-aggregator/extensions/simple-rbac/src/main/resources/META-INF/schema/examples/simple-rbac.cfg.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<simple-rbac xmlns="http://docs.openrepose.org/repose/simple-rbac/v1.0"
+             roles-header-name="X-ROLES"
+             enable-masking-403s="false">
+    <resources>
+        [<![CDATA[
+/path/to/this  GET       role1,role2,role3,role4
+/path/to/this  PUT       role1,role2,role3
+/path/to/this  POST      role1,role2
+/path/to/this  DELETE    role1
+/path/to/that  GET,PUT   ALL
```

Comment:
The CDATA block is the new RBAC DSL that is defined in REP-2097. It is basically fully qualified path [whitespace] HTTP Method(s)(CSV) [whitespace] rax:role(s)(CSV). The special wildcard words are ANY and ALL which are interchangeable, but were intended to mean ANY HTTP Method and ALL rax:roles.